README.md

```diff
@@ -1 +1,62 @@
-# Hello
\ No newline at end of file
+<h3 align="center">Medium Scraper-Ruby</h3>
```

Comment:
- [] [REQUIRED] Looks like this title doesn't fit the project. Let's make sure to update this one. It gives a good boost to the professional look of your project.